composer.json

```diff
@@ -57,7 +65,9 @@
         "2759397 - Patch EntityReferenceItemNormalizer to prevent recursion":
           "https://www.drupal.org/files/issues/2759397-1-entity_reference_recursion.patch",
         "2679775 - Fixes float issue with inline label fields (entity references on most cases).":
-          "https://www.drupal.org/files/issues/2679775-11-inline-labels.patch"
+          "https://www.drupal.org/files/issues/2679775-11-inline-labels.patch",
+        "2788777 - Allow a profile to be installed from existing config":
+          "https://www.drupal.org/files/issues/2788777-91.patch"
```

Comment:
Check 2788777.  I thought Lightning had a patch for this (lightning_core?).  We want to be sure we can use the latest patch that the Lightning folks are working on.  I believe they forked the "excluded" functionality into another issue.  Ideally we want to follow the config_installer method since that is going into core 8.6.x in Oct.

Reply:
Depending on your workflow, you might still need 2788777. We personally use the Config Installer profile.

As for profile inheritance and exclusion, Lightning with Drupal core 8.5.x still uses a single combined patch. Our branch for core 8.6.x uses patches directly from 1356276 and 2914389. But those won't apply to 8.5.x.

I hope that helps.